src/BloomExe/Edit/AudioRecording.cs

```diff
@@ -50,6 +50,14 @@ private AudioRecorder Recorder
 						_recorder = new AudioRecorder(1);
 						_recorder.PeakLevelChanged += ((s, e) => SetPeakLevel(e));
 						BeginMonitoring(); // will call this recursively; make sure _recorder has been set by now!
+						Application.ApplicationExit += (sender, args) =>
+						{
+							if (_recorder != null)
+							{
+								_recorder.Dispose();
```

Comment:
I think we should we surround that in a try/catch that swallows any unexpected errors in release builds.